src/js/containers/search/filters/location/LocationListContainer.jsx

```diff
@@ -99,13 +100,33 @@ class LocationListContainer extends React.Component {
     }
 
     render() {
+        const values = [];
```

Comment:
Can we move the parsing of autocomplete locations out of the `render()` function, which may run multiple times for reasons other than autocomplete changes? This prevents the loop from having to needlessly run; consider putting this in state instead.